docs/source/community/contributing.rst

```diff
@@ -78,3 +78,18 @@ These checks are:
 * The build must pass on Jenkins
 * The PR must be approved by at least two reviewers and there cannot be 
   outstanding changes requested
+
+**Integrating Github Commits with JIRA**
+
+You can link JIRA issues to your Hyperledger Sawtooth github commits to integrate
+the developer's activity with the associated issue. JIRA uses the issue key to
+associate the commit with an issue, so the commit can be summarized in the
+development panel for the JIRA issue.
+
+When you make a commit, add the JIRA Software issue key to the commit message 
+as follows:
+
+.. code-block:: console
+
+   $ git commit -m "PROJ-123 add a README file to the project."
```

Comment:
JIRA issue should never be on the first line (we will probably script enforcement of this).

See: https://chris.beams.io/posts/git-commit/

At the end of a commit, it's okay to have "Resolves: STL-123".

Another preferred method is to have the STL-xxx in the branch name that is pushed up.